src/utils/filter.ts

```diff
@@ -63,3 +71,21 @@ export const makeCampRoomHitFilter = ({
 
   return isWithinBounds(checkPercent, roomBounds);
 };
+
+export const filterUnreadPrivateChats = (
+  chats: WithId<PrivateChatMessage>[],
+  user: FirebaseReducer.AuthState | undefined
```

Comment:
Prefer using the `?` syntax rather than `| undefined` when possible.

```suggestion
  user?: FirebaseReducer.AuthState
```

Reply:
This is now using the optional syntax.